tests/validation/tests/v3_api/test_windows_cluster.py

```diff
@@ -8,36 +7,40 @@
 from .test_rke_cluster_provisioning import create_custom_host_from_nodes
 from .test_rke_cluster_provisioning import HOST_NAME
 
-from lib.aws import AmazonWebServices
-from threading import Thread
+from lib.aws import AmazonWebServices, AWS_DEFAULT_AMI, AWS_DEFAULT_USER
+
 
 def test_windows_provisioning():
     node_roles_linux = [["controlplane"], ["etcd"], ["worker"]]
     node_roles_windows = [["worker"], ["worker"], ["worker"]]
 
     win_nodes = \
         AmazonWebServices().create_multiple_nodes(
-            len(node_roles_windows), random_test_name(HOST_NAME), os_version="windows-1903", 
-            docker_version="19.03")
+            len(node_roles_windows), random_test_name(HOST_NAME))
 
-    aws_nodes = \
+    linux_nodes = \
         AmazonWebServices().create_multiple_nodes(
```

Comment:
Why are we calling create multiple when creating a single node?

Reply:
this is existing code but I think it is for consistency.  `create_multiple_nodes` is a wrapper function and does a little more checks than `creat_single_node`

Reply:
Ok makes sense to me, besides this wasn't changed.